tests/test_dimension.py

```diff
@@ -1215,9 +1216,12 @@ def test_topofusion_w_subdims_conddims_v2(self):
 
         # Check generated code -- expect the gsave equation to be scheduled together
         # in the same loop nest with the fsave equation
-        assert len(op._func_table) == 2
-        assert len(FindNodes(Expression).visit(op._func_table['bf0'].root)) == 3
-        assert len(FindNodes(Expression).visit(op._func_table['bf1'].root)) == 2
+        _, bns = get_blocked_nests(op, {'i0x0_blk0', 'x0_blk0'})
+        assert len(FindNodes(Expression).visit(bns['i0x0_blk0'])) == 3
+        exprs = FindNodes(Expression).visit(bns['x0_blk0'])
+        assert len(exprs) == 2
```

Comment:
NFR: The additional checks were added by me, as the already existing comment was suggesting, could be removed.

Reply:
ok